src/main/java/dk/cngroup/lentils/service/ProgressService.java

```diff
@@ -0,0 +1,35 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.entity.Cypher;
+import dk.cngroup.lentils.entity.Team;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProgressService {
+
+    private final StatusService statusService;
+
+    @Autowired
+    public ProgressService(StatusService statusService) {
+        this.statusService = statusService;
+    }
+
+    public CypherStatus setSkipProgressStatus(Cypher cypher, Team team) {
+        statusService.initializeStatusForTeamAndCypher(cypher, team);
```

Comment:
Tady se mě osobně nelíbí 2 věci. Jsou to 2 operace, které by měli podle mě běžet v transakci. Protože teď první volání nastaví sloupec na NULL a druhá metoda nastaví správnou hodnotu. Ono se to teď asi hned tak nestane, ale při selhání druhé metody by se znehodnotila hodnota ve sloupci.

Buď bych to dal do jedné metody  nebo použil nějaký UPSERT, ale to nevím jak napsat ty jo.
Nechám ještě na review od @mjanousek

Reply:
mam to teda odstranit nebo to mergneme a ja naimplementuju ten novy koncept?